test/manual/kerberos.test.ts

```diff
@@ -34,27 +34,14 @@ describe('Kerberos', function () {
     client = null;
   });
 
-  if (process.env.MONGODB_URI == null) {
-    console.error('skipping Kerberos tests, MONGODB_URI environment variable is not defined');
-    return;
-  }
   let krb5Uri = process.env.MONGODB_URI;
-  const parts = krb5Uri.split('@', 2);
-  const host = parts[1].split('/')[0];
+  const host = process.env.SASL_HOST;
 
-  if (!process.env.KRB5_PRINCIPAL) {
-    console.error('skipping Kerberos tests, KRB5_PRINCIPAL environment variable is not defined');
+  if (!process.env.PRINCIPAL) {
+    console.error('skipping Kerberos tests, PRINCIPAL environment variable is not defined');
     return;
   }
 
-  if (process.platform === 'win32') {
```

Comment:
We don't run these tests on windows

Reply:
Would this still be relevant in some way for local development?

Reply:
I don't think so - `LDAPTEST_PASSWORD` doesn't exist as an evergreen project variable, so it likely never worked.  Or, if it did at some point, it hasn't worked for years.  If we ever need to run these tests on windows, we can address this issue at that time.

Reply:
There's no special handling needed here now - the URI from the secrets manager already has the correct value regardless of OS.